src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/automatic-anomalies.mdx

```diff
@@ -17,10 +28,792 @@ Automatic anomaly detection is the easiest way for you and your team to detect a
+  5. Select `entitiesData.types` `contains` `Application`.
+  6. Select `tag.entity.guid` to select an entity of your choice.
+  7. Select the `signalType` `contains` and the signal you would like to be notified on.
+  
+      <img
+      title="Configure your workflow"
+      alt="Configure your workflow"
+      src={anomalyWorkflowFilterData}
+      />
+
+  8. Click **Additional settings** if you want to enrich your data.
+  9. Select a destination channel of your choosing.
+  
+      <img
+      title="Notifications for workflows"
+      alt="A screenshot depicting destinations for workflows"
+      src={anomalyWorkflowNotify}
+      />
+
+  10. Click **Test workflow** if you want to check if you workflow works.
+  11. Click **Activate workflow**.
+
+  To reduce noise, we recommend sending anomalies to a low priority notification channel such as Slack or email.
+
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="mute-notifications"
+    title="Mute notifications"
+    >
+
+  In Slack, you can mute temporarily or permanently detections coming from specific applications. You can also mute temporarily the entire channel. This is useful in the case of an incident or when the channel should otherwise not be interrupted.
+
+  To mute in Slack, select **Mute this app's warnings** or **Mute all warnings**, then select the duration. Notifications will resume once the muting duration has completed.
+
+  Muting an application permanently removes it from the configuration. To add it back in, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Anomaly detection**, and select the configuration to edit.
+
+  Muting anomaly detection notifications doesn't affect alerts.
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="use-anomaly-detection-Slack-messages"
+    title="Use anomaly detection Slack messages"
+    >
+
+  Each anomaly message has several key pieces of information you can use to learn more about and start troubleshooting the potential issue:
+
+  * The application name and a link to more information about it in the New Relic UI.
+  * The metric experiencing an anomaly and a link to its details in the New Relic UI.
+  * A graph of the metric over time to provide a visual understanding of the anomaly's behavior and degree.
+  * An **Analyze** button that navigates to an analysis page in applied intelligence that identifies key attributes that are unique to the anomaly, anomalies found upstream or downstream, and any other relevant signals.
+
+  Once an anomaly has returned to normal, we send a recovery notification with the option to provide feedback. Your feedback provides our development team with input to help us improve detection quality. In the case of feedback provided on throughput anomalies, an evaluation is run each hour based on feedback to fit a more suitable model. If we helped you, you can select **Yes** or **No**.
```

Comment:
```suggestion
  Once an anomaly has returned to normal, we send a recovery notification with the option to provide feedback. Your feedback provides our development team with input to help us improve detection quality. When feedback is provided on throughput anomalies, an evaluation is run each hour based on your feedback to find a more suitable model. If we helped you, you can select **Yes** or **No**.
```

Reply:
@nbaenam these changes could have made the statement incorrect! Don't incorporate them if I made them wrong.